tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -103,13 +103,13 @@ impl std::ops::Mul<usize> for NoiseLevel {
 }
 
 impl Degree {
-    pub(crate) fn after_bitxor(&self, other: Self) -> Self {
+    pub(crate) fn after_bitxor(self, other: Self) -> Self {
         let max = cmp::max(self.0, other.0);
         let min = cmp::min(self.0, other.0);
         let mut result = max;
 
         //Try every possibility to find the worst case
-        for i in 0..min + 1 {
+        for i in 0..=min {
```

Comment:
in't there seomthing about ..= being slower than .. ? likely does not really matter in our case but I seem to remember a quirk around that

Reply:
https://old.reddit.com/r/rust/comments/15tvuio/why_isnt_the_for_loop_optimized_better_in_this/

Reply:
yeah that's what I seem to rememeber, if min is not supposed to be usize::MAX I think we can keep the range without the =

Reply:
Unfixed